ci-operator/config/openshift/kubernetes/openshift-kubernetes-master__okd-scos.yaml

```diff
@@ -7,11 +23,13 @@ base_images:
     name: scos-4.16
     namespace: origin
     tag: base-stream9
+binary_build_commands: openshift-hack/build-go.sh
 build_root:
   image_stream_tag:
     name: builder
     namespace: ocp
-    tag: rhel-8-golang-1.21-openshift-4.16
+    tag: rhel-9-golang-1.21-openshift-4.16
```

Comment:
k8s master is being built with go 1.22 nowadays, so this will fail for you https://github.com/openshift/kubernetes/blob/master/.ci-operator.yaml#L4. I'd suggest using the:
```
build_root:
  from_repository: true
```
instead.